lib/std/crypto/Certificate/Bundle/macos.zig

```diff
@@ -61,10 +61,16 @@ pub fn rescanMac(cb: *Bundle, gpa: Allocator) RescanMacError!void {
             }
 
             for (record_list) |record_offset| {
+                // An offset of zero means that the record is not present.
+                // An offset that is not 4-byte-aligned is invalid.
+                if (record_offset == 0 or record_offset % 4 != 0) continue;
```

Comment:
I'm not fully sure about the `record_offset % 4 != 0` part of this. It is based on:

- https://github.com/n0fate/chainbreaker/blob/2eb467d9355e61f1dfecee1cec41b06033186326/chainbreaker/__init__.py#L280
- https://gist.github.com/williballenthin/d6bf9f1553d9fa27e0cc6880a6d992b4#file-macos_keychain-py-L691 (this one just checks if the LSB is set, so not exactly the same check)